docs/en/advanced/nn_module_utilities.md

```diff
@@ -0,0 +1,40 @@
+# Neural Network Module Utilities
+
+This page will explain the parts of the module that do not directly concern something with the neural network, but PX4 related implementations, so that you easily can shape the module to your needs.
```

Comment:
What module? I mean I know what module, but someone might come to this page by searching. 
So something like 
```suggestion
This page explains the parts of the [name-of-module module](link to module) that do not directly concern something with the neural network, but PX4 related implementations, so that you easily can shape the module to your needs.
```

Reply:
YOu can refer to it as "the module" after the first definition